pkg/cmd/cmd.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	versionCmd "github.com/deepsourcelabs/cli/pkg/cmd/version"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdRoot(buildVersion, buildDate string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deepsource <command> <subcommand> [flags]",
+		Short: "DeepSource CLI",
+		Long:  `Now, ship good code directly from command line.`,
+	}
+	cmd.AddCommand(versionCmd.NewCmdVersion(buildVersion, buildDate))
+
+	return cmd
+}
+
+func Execute(buildVersion, buildDate string) {
+	cmd := NewCmdRoot(buildVersion, buildDate)
+	if err := cmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
Can't we terminate the error in the calling function.

Reply:
Ack

Reply:
Done. PTAL